apps/etterlatte-tidshendelser/src/main/kotlin/no/nav/etterlatte/tidshendelser/HendelsePoller.kt

```diff
@@ -0,0 +1,77 @@
+package no.nav.etterlatte.tidshendelser
+
+import no.nav.etterlatte.jobs.LoggerInfo
+import no.nav.etterlatte.jobs.fixedRateCancellableTimer
+import no.nav.etterlatte.libs.common.logging.getCorrelationId
+import no.nav.etterlatte.libs.common.logging.sikkerlogger
+import no.nav.etterlatte.libs.common.logging.withLogContext
+import no.nav.etterlatte.libs.jobs.LeaderElection
+import org.slf4j.LoggerFactory
+import java.time.Clock
+import java.time.Duration
+import java.util.Date
+import java.util.Timer
+
+class HendelsePollerTask(
+    private val leaderElection: LeaderElection,
+    private val initialDelaySeconds: Long,
+    private val periode: Duration,
+    private val clock: Clock,
+    private val hendelsePoller: HendelsePoller,
+    private val maxAntallHendelsePerPoll: Int = 5,
+) {
+    private val logger = LoggerFactory.getLogger(HendelsePollerTask::class.java)
+
+    fun start(): Timer {
+        logger.info("Starter polling av nye hendelser pr $periode")
+
+        return fixedRateCancellableTimer(
+            name = "HENDELSE_POLLER_TASK",
+            startAt = Date(clock.millis() + (initialDelaySeconds * 1000)),
```

Comment:
Litt rart at denne tar inn java.util.Date og millisekund for periode, eigentleg. Bør oppgradere til LocalDate/Tidspunkt og Duration/Period etter kvart.

Reply:
Har skrevet om til en variant som tar inn `initialDelay` direkte, i stedet for `startAt`. Både Date og Clock faller da vekk.